### DISCUSSION TITLE
Add go2rtc link to the Web Server

### DISCUSSION THREAD

**Discussion by Vendo232** [gtxaspec/wz_mini_hacks#603] [General]
It would be great to add go2rtc link http://<camera_ip>:1984 to the web server page @virmaior

**Comment by virmaior** [comment]
will do when I get a bit more time!
one complication is that the base page is static rather than sh-based...
but I've got an idea to make things better on that front.

**Comment by virmaior** [comment]
https://github.com/gtxaspec/wz_mini_hacks/pull/606

**Comment by Vendo232** [comment]
Thank you !
